### PR TITLE
Grunt async is marked as done, before getCoverageData is finished

### PR DESCRIPTION
**Pull request by mdasberg**
see https://github.com/r3b/grunt-protractor-coverage/issues/60

### CODE REVIEW

File: tasks/protractor_coverage.js [hunk 1]
```diff
@@ -103,11 +103,11 @@ module.exports = function(grunt) {
     coverageDir = coverageDir.replace(/\\/g,'/');
     var noInject = opts.noInject;
     if (!noInject) {
-      var saveCoverageTemplate = grunt.file.expand([ opts.saveCoverageTemplate, "node_modules/grunt-protractor-coverage/resources/saveCoverage.tmpl", process.cwd()+'/**/resources/saveCoverage.tmpl']).shift();
```
Comment (r3b) [inline tasks/protractor_coverage.js]: Why change the var names? That template doesn't contain anything related to cucumber...

Comment (mdasberg) [inline tasks/protractor_coverage.js]: The current build breaks on jshint errors:

```js
[L106:C32] 'saveCoverageTemplate' is already defined.
      var saveCoverageTemplate = grunt.file.expand([ opts.saveCoverageTemplate, "node_modules/grunt-protractor-coverage/resources/saveCoverage.tmpl", process.cwd()+'/**/resources/saveCoverage.tmpl']).shift();
```
https://travis-ci.org/r3b/grunt-protractor-coverage/builds/112179951

Comment (mdasberg) [inline tasks/protractor_coverage.js]: You are right the name should not contain cucumber. noInject is used for cucumber, this should definitely not be called that. I will rename it.

Comment (mdasberg) [inline tasks/protractor_coverage.js]: Should be fixed now with the latest commit

**Comment by r3b** [comment]
Thanks!

**Pull request merged**